test/type/string_test.exs

```diff
@@ -0,0 +1,107 @@
+defmodule Litmus.Type.StringTest do
+  use ExUnit.Case
+
+  alias Litmus.Type
+
+  test "tests protocol implementation for String data type" do
+    type = %Type.String{
+      required: true
+    }
+
+    field = :id
+
+    data = %{
```

Comment:
if you make this a one-liner, will the formatter force this into multiple lines? i.e. `data = %{id: "1"}`

Reply:
It doesn't. Ill change them